Makefile

```diff
@@ -51,8 +51,14 @@ KERNEL_REL := $(shell uname -r)
 export KERNEL_REL
 
 # Sysbox image-generation globals utilized during the testing of sysbox installer.
-IMAGE_BASE_DISTRO := $(shell lsb_release -ds | cut -d' ' -f1 | tr '[:upper:]' '[:lower:]')
-IMAGE_BASE_RELEASE := $(shell lsb_release -cs)
+IMAGE_BASE_DISTRO := $(shell lsb_release -is | tr '[:upper:]' '[:lower:]')
+ifeq ($(IMAGE_BASE_DISTRO), ubuntu)
+	IMAGE_BASE_RELEASE := $(shell lsb_release -cs)
+else
```

Comment:
Should we flip this around? If centOS, do the action specific to it; else do the action for ubuntu.

Is CenOS the outlier or Ubuntu the outlier regarding figuring out the image base release?

Reply:
CentOS is definitely more of an "outlier" than Ubuntu in this case, but every new distro that we'll support has a similar non-conventional heuristic, so eventually, Ubuntu will be the outlier (i.e. we will need a bunch of 'else if'); that's why i thought of leaving ubuntu at the top as our most popular distro.

However, till we get there, your point is valid so will change ;-)